src/Types.ts

```diff
@@ -19,6 +19,7 @@ export interface TTokenRequestForRefresh extends TTokenRqBase {
 export type TTokenRequest = TTokenRequestWithCodeAndVerifier | TTokenRequestForRefresh
 
 export type TTokenData = {
+  // biome-ignore lint: reason
   [x: string]: any
 }
```

Comment:
Didn't it allow the `any`?

Btw, I think the "reason" is something you can replace with your reason for the ignore 😅